example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl

```diff
@@ -0,0 +1,43 @@
+@prefix area: <https://ontology.okp4.space/thesaurus/area/> .
+@prefix core: <https://ontology.okp4.space/core/> .
+@prefix dataset: <https://ontology.okp4.space/dataverse/dataset/> .
+@prefix dataspace: <https://ontology.okp4.space/dataverse/dataspace/> .
+@prefix dstmeta: <https://ontology.okp4.space/metadata/dataset/> .
+@prefix license: <https://ontology.okp4.space/thesaurus/license/> .
+@prefix mediatype: <https://ontology.okp4.space/thesaurus/media-type/> .
+@prefix meta: <https://ontology.okp4.space/metadata/> .
+@prefix service: <https://ontology.okp4.space/dataverse/service/> .
+@prefix topic: <https://ontology.okp4.space/thesaurus/topic/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+meta:8de7ca33-378e-445c-b087-f5ef334d2cca a dstmeta:GeneralMetadata ;
+    core:describes dataset:0ea1fc7a-dd97-4adc-a10e-169c6597bcde ;
+    core:hasCreator "Anses"^^xsd:string ;
```

Comment:
I'd remove the `xsd:string` type annotation. According to the [RDF specification](https://www.w3.org/TR/turtle/#turtle-literals), if a literal value in RDF does not have an explicit datatype IRI or a language tag, then the datatype is assumed to be `xsd:string` by default.